pkg/GAPJulia/JuliaInterface/julia/gap.jl

```diff
@@ -44,6 +44,8 @@ Base.setproperty!(x::MPtr, f::Union{AbstractString,Int64}, v) = Globals.ASS_REC(
 Base.zero(x::GAPInputType_internal) = Globals.ZERO(x)
 Base.one(x::GAPInputType_internal) = Globals.ONE(x)
 Base.:-(x::GAPInputType_internal) = Globals.AINV(x)
+Base.:*(x::GAP.MPtr,y::String) = x*GAP.julia_to_gap(y)
+Base.getindex(x::GAP.MPtr,y::String) = Globals.ELM_LIST(x,GAP.julia_to_gap(y))
```

Comment:
Hmm... That's quite specific, and would break our rule of avoiding implicit conversion unless they are free, or absolutely required.

All in all, wouldn't it make more sense to put these conversions into a dedicated `Homalg.jl`?

Reply:
Yes, maybe. I have no objection in starting a `homalg.jl` :)